src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,93 @@
+/* = Navigation
+============================ */
+.tribe-events {
```

Comment:
Does this make it hard to reuse the classes in other plugins?

Reply:
We're actually using it like that already on the other components. We'll be probably moving out these into their own repo. I think events is the only one with views and maybe that's the reason why we ended up going that way https://github.com/moderntribe/tribe-common/blob/release/B19.06/src/resources/postcss/components/_search.pcss#L1

Maybe @bordoni or @paulmskim have another answer for this question/

Reply:
If it's prefixed with .tribe-events, it ought to be put into TEC styles?

Reply:
Everything in the kitchen sink is  prefixed with .tribe-events

For example:

- https://github.com/moderntribe/tribe-common/blob/release/B19.06/src/resources/postcss/base/typography/_body.pcss
- https://github.com/moderntribe/tribe-common/blob/release/B19.06/src/resources/postcss/base/typography/_ctas.pcss
- https://github.com/moderntribe/tribe-common/blob/release/B19.06/src/resources/postcss/base/typography/_headings.pcss

We're reseting many html things, and ideally we'll move these pcss files to their own repo that will be included on the other plugins -> https://github.com/moderntribe/tribe-common-styles

Reply:
I just think it's odd anything plugin-specific should end up in common/common-styles. If everyone is OK with this then that's fine for me, just want to bring it up :)

Reply:
TEC is the only one that has views. We'll eventually have a separate repo for this (the one I linked above) to store some variables, mixins, etc. This is a WIP, so until we have the separate repo working as a module of the plugins, we're having this in common. 

Let's wait for @bordoni or @paulmskim to chime in.

Reply:
All the styles (resets, components, base, etc.) will have a wrapper of `.tribe-events`. Though we define our own classes (`.tribe-common-h1`, for example), there is the small chance we clash and override a customer's styles. We don't want that. Also in the reset, we don't want to reset customer elements. So we wrap all styles in the wrapper class. Each component, block, page, etc. that is defined by TEC will have this setup. If we extend this views work to tickets, for example, we will also add `.tribe-tickets` wrapper class.

Reply:
@juanfra let's change this to `.tribe-common` as per our earlier discussion